proxy/http/HttpTransact.h

```diff
@@ -1149,23 +1149,31 @@ class HttpTransact
 
 typedef void (*TransactEntryFunc_t)(HttpTransact::State *s);
 
+////////////////////////////////////////////////////////
+// the spec says about message body the following:    //
+// All responses to the HEAD request method MUST NOT  //
+// include a message-body, even though the presence   //
+// of entity-header fields might lead one to believe  //
+// they do. All 1xx (informational), 204 (no content),//
+// and 304 (not modified) responses MUST NOT include  //
+// a message-body.                                    //
+////////////////////////////////////////////////////////
 inline bool
-is_response_body_precluded(HTTPStatus status_code, int method)
+is_response_body_precluded(HTTPStatus status_code)
```

Comment:
Are you asking why we need `is_response_body_precluded(HTTPStatus status_code)` instead of `is_response_body_precluded(HTTPStatus status_code, int method)`?